default/scripting/techs/spy/DETECT_5.focs.py

```diff
@@ -16,7 +16,7 @@
         ),
         EffectsGroup(
             scope=Source,
-            effects=SetEmpireMeter(empire=Source.Owner, meter="METER_DETECTION_STRENGTH", value=Value + 200),
+            effects=SetEmpireMeter(empire=Source.Owner, meter="METER_DETECTION_STRENGTH", value=Value + 300),
```

Comment:
based on https://www.freeorion.org/forum/viewtopic.php?p=106298#p106298

@Oberlus any comment / objections?

Reply:
The linked calculations were a bit off (as I commented in that thread):

Multispectral shield does not stack with max cloack part, so the maximum figures there need to subtract 60, down to 215 in an Ion Storm with owned Resonant Moon and that GG-cloacking part of LienRag. Without the GG-cloacking part maximum is 185 with Ion Storm. I am also not charmed by the GG-cloacking part. Allowing all kind of hulls to hide in GGs, that are quite commons, seems to me a bad idea. Not so "situational".

Since Ion Storms are temporal, I don't see the need to change maximum from 200 to 300. I think it is good for gameplay to allow temporal cloackings at maximum detection strength. More so if one accounts for the decreasing stealth parts/buildings that can make it impossible even with Ion Storms.

All in all, I would not merge this.